### ISSUE TITLE
Issue Field missing timetracking

### ISSUE THREAD

**Issue by Urlak** [dapplo/Dapplo.Jira#10]
Fields don't have "timetracking" ,"originalEstimate" "remainingEstimate".
Example:
  "timetracking":
        {
           "originalEstimate": "1d 2h",
           "remainingEstimate": "3h 25m"
        }
It is not possible to add "originalEstimate" and "remainingEstimate".

https://developer.atlassian.com/jiradev/jira-apis/jira-rest-apis/jira-rest-api-tutorials/jira-rest-api-example-create-issue#JIRARESTAPIExample-CreateIssue-Addingaworklogentryduringcreate

**Comment by Lakritzator** [comment]
Thanks for reporting this!
I made changes to 0.5.14 which is available on NuGet as we speak.

I don't have tests for creating issues yet, but the information is at least read correctly with client.Issue.GetAsync in the test cases on my ci server.

Let me know if it works for you!

This week I will spend some time to set-up a testing project in my Jira, so my tests can create tickets without irritating my colleagues.

P.S.
I would really like some feedback on my project, like for what you are using it and what you like and or dislike.

**Comment by Lakritzator** [comment]
With 0.5.16 I also added conversion from TimeSpan to Jira time notation (1w 2d 4h 5m) and back again. An example can be found in the TimeSpanTests test case:
https://github.com/dapplo/Dapplo.Jira/blob/master/Dapplo.Jira.Tests/TimeSpanTests.cs

I also added, to support this, a way to get the configuration for the calculation from the jira server as some people might have less or more working hours and days. Default is 5 working days and 8 working hours a day.
